source/source.go

```diff
@@ -74,6 +74,9 @@ type Service interface {
 	// Config defines a function that captures
 	// the pipeline configuration from a repo.
 	Config(user *library.User, org, name, ref string) ([]byte, error)
+	// ConfigBackoff defines a function wrapper that
+	// checks if yml/yaml file is valid.
```

Comment:
We might want to think about adding wording noting the frequency or type (Exponential, Constant, Linear, etc) of backoff we do.

Reply:
ConfigBackOff is a "truncated constant" backoff and is called five times every five seconds.